src/ai/lua/core.cpp

```diff
@@ -498,6 +498,14 @@ static int cfun_ai_get_recruitment_pattern(lua_State *L)
 	return 1;
 }
 
+static int cfun_ai_get_retreat_factor(lua_State *L)
```

Comment:
I would've suggested omitting this, since these are supposed to be deprecated and this one even _is_ deprecated...

Reply:
Taken care of in 200f5dd913